torchvision/transforms/transforms.py

```diff
@@ -1400,35 +1413,46 @@ def get_params(img, scale, ratio, value=0):
         area = img_h * img_w
 
         for _ in range(10):
-            erase_area = random.uniform(scale[0], scale[1]) * area
-            aspect_ratio = random.uniform(ratio[0], ratio[1])
+            erase_area = area * torch.empty(1).uniform_(scale[0], scale[1]).item()
+            aspect_ratio = torch.empty(1).uniform_(ratio[0], ratio[1]).item()
 
             h = int(round(math.sqrt(erase_area * aspect_ratio)))
             w = int(round(math.sqrt(erase_area / aspect_ratio)))
+            if value is None:
+                v = torch.empty([img_c, h, w], dtype=torch.float32).normal_()
+            elif len(value) == 1:
+                v = torch.tensor(value[0])
+            else:
+                v = torch.tensor(value, dtype=torch.float32).view(-1, 1, 1).expand(-1, h, w)
```

Comment:
nit: this was a small performance optimization, as before we would only compute `v` if the currently sampled `h, w` were valid. This is more important when `value is None`, because we generate random tensors of potentially large sizes.

Reply:
nit for a follow-up PR:  I think there are two cases that can be fused together: when `len(value) == 1` and `len(value) == img_c`, they can both be combined as:
```python
v = torch.tensor(value)[:, None, None]
```
I don't think we need to force the data to be `float32` anymore, because we have automatic type promotion now in PyTorch.

But as I said, this micro-simplification is not for this PR, as ideally we want to keep things as close as possible to the original implementation for this port.